pages/mesosphere/dcos/services/edge-lb/1.5/tutorials/config-edgelb-for-k8s/index.md

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
Users can also use auto pools for this now. Ref: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kubernetes-cluster/1700/config.json#L268-L303

Reply:
Created separate Jira ticket

Reply:
Created a separate ticket.